app/src/main/java/com/chatgptlite/wanted/constants/Constants.kt

```diff
@@ -8,4 +8,4 @@ const val urlToAvatarGPT = "https://gptapk.com/wp-content/uploads/2023/02/chatgp
 const val urlToGithub = "https://github.com/lambiengcode"
 
 const val matchResultString = "\"text\":"
-const val matchResultTurboString = "\"content\":"
\ No newline at end of file
+const val matchResultTurboString = "\"content\":"
```

Comment:
what is different here?

Reply:
Sorry for the confusion my commit not clear. I have stored a val in here and made a push but found out it is not a good idea and removed it that is why noting has been changed in here.